bot/cogs/moderation/modlog.py

```diff
@@ -98,7 +98,10 @@ async def send_log_message(
         footer: t.Optional[str] = None,
     ) -> Context:
         """Generate log embed and send to logging channel."""
-        embed = discord.Embed(description=text)
+        # Truncate string directly here to avoid removing newlines
+        embed = discord.Embed(
+            description=text[:2046] + "..." if len(text) > 2048 else text
```

Comment:
This will create a 2049 length string, if it's above 2048

Reply:
Yeah, I'm a little suspicious of these hardcoded values too. Can you justify them? Is there a test for this particular part of the code? Have you tested this manually?

Reply:
I will fix this length issue, but I can't change this. Otherwise, we are getting [this issue](https://github.com/python-discord/bot/pull/864#pullrequestreview-390849141). This already tests it [here](https://github.com/ks129/bot/blob/ban-kick-reason-length/tests/bot/cogs/moderation/test_modlog.py#L17), but I have same issue there too.

Reply:
That's fine, just make sure the values are correct. It's mostly this length issue I was concerned about.